native/run_atlas.sh

```diff
@@ -62,6 +62,7 @@ else
     cleanexit 1
   fi
 fi
+echo "$cvmfs_config_stat"
```

Comment:
Maybe this should be moved to line 60 directly after the stat command is run? Then the output would be logged even if the command failed.

Reply:
The test in L60 now checks for the exit code of "cvmfs_config" in L59.
If you move the "echo ..." up between L59 and L60 the test must also be modified.
Otherwise it would check for the exit code of "echo ...".

Since there is already an error message at L61 and since the output is only used to manually identify weird settings I placed it at L65.